src/util/RouterUtil.js

```diff
@@ -100,18 +100,18 @@ function (Okta, Util, OAuth2Util, Enums, BrowserFeatures, Errors, ErrorCodes) {
 
     switch (res.status) {
     case 'SUCCESS':
```

Comment:
Can we add a comment indicating that the recovery type check should be the first one made?  That might help us avoid another issue like this.